docs/research/power-consumption-tests.md

```diff
@@ -0,0 +1,160 @@
+# PC Engines apu3 power consumption tests
+
+We have started research and measurements of power consumption on apu platforms.
+Our goal is to optimize (if possible) power management on coreboot level. First
+stage of work is to collect data from various tests and create reports. For that
+purpose we have used RTE with power monitor sensor.
+
+This document gathers all already carried out tests, their results and our
+conclusions. Over time, we will conduct next tests and hence extend this
+document with further reports.
+
+## Tests
+
+### apu3 stress-test cpu
+
+Test case is following:
+
+1. Power on RTE and power on apu3.
+2. Boot to Debian on apu3.
```

Comment:
@piotr-kleins which Debian, which kernel

Reply:
@miczyg1 @macpijan this brings back topic of `meta-pcengines` and system with some optimizations in whole stack e.g.:

1. Network performance-optimized image
2. Power consumption-optimized image
3. Security-optimized image

This PC Engines repository should be source of truth for correct configuration. I know this is big topic, that's why I would expect it to be divided and splitter over 12 months or so. @miczyg1 please follow up with internal project discovery procedure for above items. This note doesn't block progress in power consumption optimization.

Reply:
Added Debian and kernel version

Reply:
@macpijan @miczyg1 please confirm that you read that and will proceed with the effort to make this happen.

Reply:
Created a JIRA story for that